ipaserver/plugins/dogtag.py

```diff
@@ -1680,47 +1190,80 @@ def take_certificate_off_hold(self, serial_number):
         +---------------+---------------+---------------+
         |result name    |result type    |comments       |
         +===============+===============+===============+
-        |unrevoked      |bool           |               |
+        |requestStatus  |bool           |               |
         +---------------+---------------+---------------+
-        |error_string   |unicode        |               |
+        |errorMessage   |unicode        |               |
         +---------------+---------------+---------------+
+
+        The REST API responds with JSON in the form of:
+
+        {
+          "requestID":"0x17",
+          "requestType":"revocation",
```

Comment:
Should be "requestType":"**un**revocation"

Reply:
I fixed this and added some more language around how this works. I removed the ack so you can double-check my updated wording.

Reply:
Thanks, looks good to me. I'm re-adding the ack (and rerunning azure)